### PR TITLE
Xero Partner updates

### PR DESCRIPTION
**Pull request by pzaichkina**
# Description

Changes to Xero App Partnership and Xero Bank Feeds Partner pages.

## Type of change

Please delete options that are not relevant.

- [X] New document(s)/updating existing
- [ ] Fixes
- [ ] Styling
- [ ] Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

File: docs/integrations/accounting/xero/xero-app-partner-program.md [hunk 1]
```diff
@@ -1,230 +1,38 @@
...
+title: "Become a Xero App Partner"
 description: "Unlock the full value of connectivity with Xero by becoming a Xero App Partner"
 ---
 
-import LocalStorageChecklist from "@components/global/LocalStorageChecklist";
-import BrowserOnly from "@docusaurus/BrowserOnly";
+Once you have set up your Xero integration, you should consider becoming a [Xero App Partner](https://developer.xero.com/documentation/xero-app-store/app-partner-guides/overview). This is an essential step if you want to use your Xero integration at scale. 
 
-If you want to have more than 25 Xero connections, you'll need to join the **Xero App Partner Program**.
+**Xero App Partner can:**
```
Comment (mcclowes) [inline docs/integrations/accounting/xero/xero-app-partner-program.md:8]:
```suggestion
**Xero App Partners can:**
```

File: docs/integrations/accounting/xero/xero-app-partner-program.md [hunk 2]
```diff
@@ -1,230 +1,38 @@
...
 
-Becoming a certified Xero App Partner brings the following benefits:
+- Exceed the 25 company connection cap that Xero puts on uncertified apps
+- Reach millions of Xero subscribers and advisors via the [Xero App Store](http://www.xero.com/marketplace)
```
Comment (mcclowes) [inline docs/integrations/accounting/xero/xero-app-partner-program.md:11]:
```suggestion
- Reach millions of Xero subscribers and advisors on the [Xero App Store](http://www.xero.com/marketplace)
```

File: docs/integrations/accounting/xero/xero-app-partner-program.md [hunk 3]
```diff
@@ -1,230 +1,38 @@
...
 
-To join the program, there are a series of requirements your integration must first satisfy. There are also some additional use case-specific requirements, as explained in [Xero App Partner guidance by use case](/integrations/accounting/xero/accounting-xero-app-partner-guidance).
+We recommend speaking with your Codat account team before you decide when to apply for partnership. When you are ready to start, we will introduce you to Xero’s Partnerships team. 
 
-We've broken down the requirements you need to action based on the [Xero Certification Checkpoints](https://developer.xero.com/documentation/xero-app-store/app-partner-guides/certification-checkpoints/#required-for-all-integrations). You can use the answers on this page to help with your App Partner Program application. [We take care of a number of checkpoints for you](/integrations/accounting/xero/xero-app-partner-program#we-take-care-of).
+**Types of App Partner**
 
-:::caution Use cases
+- Financial Services Partners (eg. Bank Feeds, Lending, and Payments use cases)
```
Comment (mcclowes) [inline docs/integrations/accounting/xero/xero-app-partner-program.md:19]:
```suggestion
- Financial Services Partners (e.g. Bank Feeds, Lending, and Payments use cases)
```

File: docs/integrations/accounting/xero/xero-app-partner-program.md [hunk 4]
```diff
@@ -1,230 +1,38 @@
...
 
-For tips and advice, read our blog post on [How to get your app certified on the Xero App Store](https://www.codat.io/blog/get-your-app-certified-on-the-xero-app-store/).
-</details>
+Xero’s technical requirements can be complex and vary by use case. Codat’s products take care of many of Xero’s requirements for you, but there are some that require you to take action. 
 
-## Start your partnership application
+We recommend working closely with the Codat team to ensure Xero’s technical requirements are addressed during your implementation. 
 
-To start the Xero App Partner application process, follow these steps:
+**Xero’s commercial framework**
```
Comment (mcclowes) [inline docs/integrations/accounting/xero/xero-app-partner-program.md:36]:
```suggestion
#### Xero’s commercial framework
```

File: docs/integrations/accounting/xero/xero-app-partner-program.md [hunk 5]
```diff
@@ -1,230 +1,38 @@
...
+**Steps required to become an App Partner**
 
-<details>
-<summary><b>Additional resources</b></summary>
+- Submit an application
+- Sign Xero’s Partner Agreement
+- Pass a technical certification process
 
-In the Xero developer documentation, see [Steps to becoming an app partner](https://developer.xero.com/documentation/xero-app-store/app-partner-guides/app-partner-steps) and the [App Partner Program FAQs](https://developer.xero.com/documentation/xero-app-store/app-partner-guides/faqs/).
+**Xero’s technical requirements**
```
Comment (mcclowes) [inline docs/integrations/accounting/xero/xero-app-partner-program.md:30]:
```suggestion
#### Xero’s technical requirements
```

File: docs/integrations/accounting/xero/xero-app-partner-program.md [hunk 6]
```diff
@@ -1,230 +1,38 @@
...
 
-:::caution Use cases
+- Financial Services Partners (eg. Bank Feeds, Lending, and Payments use cases)
+- App Store Partners (all other use cases)
 
-There are additional checkpoints based on use cases.
+Some Financial Services use cases are not eligible for Xero Partnership. Speak to your Codat account team for guidance on how Xero categorizes your use case. 
 
-:::
+**Steps required to become an App Partner**
```
Comment (mcclowes) [inline docs/integrations/accounting/xero/xero-app-partner-program.md:24]:
```suggestion
#### Steps required to become an App Partner
```

File: docs/integrations/accounting/xero/xero-app-partner-program.md [hunk 7]
```diff
@@ -1,230 +1,38 @@
...
+It can take several months to become a Xero App Partner, so we advise you to begin the process well before you need to connect more than 25 Xero users via your integration. 
 
-To join the program, there are a series of requirements your integration must first satisfy. There are also some additional use case-specific requirements, as explained in [Xero App Partner guidance by use case](/integrations/accounting/xero/accounting-xero-app-partner-guidance).
+We recommend speaking with your Codat account team before you decide when to apply for partnership. When you are ready to start, we will introduce you to Xero’s Partnerships team. 
 
-We've broken down the requirements you need to action based on the [Xero Certification Checkpoints](https://developer.xero.com/documentation/xero-app-store/app-partner-guides/certification-checkpoints/#required-for-all-integrations). You can use the answers on this page to help with your App Partner Program application. [We take care of a number of checkpoints for you](/integrations/accounting/xero/xero-app-partner-program#we-take-care-of).
+**Types of App Partner**
```
Comment (mcclowes) [inline docs/integrations/accounting/xero/xero-app-partner-program.md:17]:
```suggestion
#### Types of App Partner
```

File: docs/integrations/accounting/xero/xero-app-partner-program.md (hunk 1 above)
Comment (mcclowes) [inline docs/integrations/accounting/xero/xero-app-partner-program.md:8]:
```suggestion
#### Xero App Partner can:
```

**Review by mcclowes — COMMENTED** [review]
I think there's room for a simpler call to action to talk to your Codat account team, rather than the somewhat duplicative 5 ctas ending each subsection

**Review by mcclowes — APPROVED** [review]

File: docs/integrations/accounting/xero/xero-app-partner-program.md [hunk 8]
```diff
@@ -1,230 +1,38 @@
...
-Becoming a certified Xero App Partner brings the following benefits:
+- Exceed the 25 company connection cap that Xero puts on uncertified apps
+- Reach millions of Xero subscribers and advisors on the [Xero App Store](http://www.xero.com/marketplace)
 
-- No limit to the number of companies (SMBs) you can connect to via your Xero integration.
-- The opportunity to reach millions of subscribers and advisors through the [Xero App Store](http://www.xero.com/marketplace) and [app partner program](https://developer.xero.com/app-partner).
+It can take several months to become a Xero App Partner, so we advise you to begin the process well before you need to connect more than 25 Xero users via your integration. 
 
-To join the program, there are a series of requirements your integration must first satisfy. There are also some additional use case-specific requirements, as explained in [Xero App Partner guidance by use case](/integrations/accounting/xero/accounting-xero-app-partner-guidance).
+We recommend speaking with your Codat account team before you apply for partnership. When you are ready to start, we will introduce you to Xero’s Partnerships team.
```
Comment (dcoplowe) [inline docs/integrations/accounting/xero/xero-app-partner-program.md:15]:
```suggestion
We recommend speaking with your account manager at Codat before you apply for partnership. When you are ready to start, we will introduce you to Xero’s Partnerships team. 
```

File: docs/bank-feeds-api/xero-bank-feeds/xero-bank-feeds-partner.md [hunk 9]
```diff
@@ -0,0 +1,17 @@
...
+Please speak to your Codat account team to initiate this process. It will involve:
+
+- An introduction to Xero’s Financial Services Partnerships team
+- Submitting a Bank Feeds Integration Proposal
+- Signing Xero’s Bank Feeds Partnership Agreement (non-commercial)
+- Passing a technical certification process
+
+#### Xero’s technical requirements
+
+Xero’s technical requirements are complex, but Codat’s Bank Feeds API take care of many of them for you. We recommend working closely with the Codat team to ensure you address all of Xero’s requirements during your implementation.
```
Comment (dcoplowe) [inline docs/bank-feeds-api/xero-bank-feeds/xero-bank-feeds-partner.md:17]:
```suggestion
Xero’s technical requirements are complex, but Codat’s Bank Feeds API take care of many of them for you. We recommend working closely with Codat to ensure you address all of Xero’s requirements during your implementation.
```

File: docs/integrations/accounting/xero/xero-app-partner-program.md [hunk 10]
```diff
@@ -1,230 +1,38 @@
...
 
-We've broken down the requirements you need to action based on the [Xero Certification Checkpoints](https://developer.xero.com/documentation/xero-app-store/app-partner-guides/certification-checkpoints/#required-for-all-integrations). You can use the answers on this page to help with your App Partner Program application. [We take care of a number of checkpoints for you](/integrations/accounting/xero/xero-app-partner-program#we-take-care-of).
+#### Types of App Partners
 
-:::caution Use cases
+- Financial Services Partners (e.g. Bank Feeds, Lending, and Payments use cases)
+- App Store Partners (all other use cases)
 
-There are additional checkpoints based on use cases.
+Some Financial Services use cases are not eligible for Xero Partnership. Speak to your Codat account team for guidance on how Xero categorizes your use case.
```
Comment (dcoplowe) [inline docs/integrations/accounting/xero/xero-app-partner-program.md:22]:
```suggestion
Some Financial Services use cases are not eligible for Xero Partnership. Speak to your account manager for guidance on how Xero categorizes your use case. 
```

File: docs/integrations/accounting/xero/xero-app-partner-program.md [hunk 11]
```diff
@@ -1,230 +1,38 @@
...
 
-In the Xero developer documentation, see [Steps to becoming an app partner](https://developer.xero.com/documentation/xero-app-store/app-partner-guides/app-partner-steps) and the [App Partner Program FAQs](https://developer.xero.com/documentation/xero-app-store/app-partner-guides/faqs/).
+#### Xero’s technical requirements
 
-For tips and advice, read our blog post on [How to get your app certified on the Xero App Store](https://www.codat.io/blog/get-your-app-certified-on-the-xero-app-store/).
-</details>
+Xero’s technical requirements can be complex and vary by use case. Codat’s products take care of many of Xero’s requirements for you, but there are some that require you to take action. 
 
-## Start your partnership application
+We recommend working closely with the Codat team to ensure Xero’s technical requirements are addressed during your implementation.
```
Comment (dcoplowe) [inline docs/integrations/accounting/xero/xero-app-partner-program.md:34]:
```suggestion
We recommend working closely with Codat to ensure Xero’s technical requirements are addressed during your implementation. 
```

File: docs/integrations/accounting/xero/xero-app-partner-program.md [hunk 12]
```diff
@@ -1,230 +1,38 @@
...
-Codat has implemented paginated fetching.
-
-#### 11. Webhooks (Required if getting data)
-
-Codat surfaces a select number of events from Xero based on Codat metadata (e.g. if a company within Codat has a connection status change).
-
-#### 14. Rounding (Required if posting or putting data)
-
-Codat handles the rounding of totals and ensuring they match what’s in Xero.
+Xero may have a fee structure for your use case. For guidance on Xero fees, please speak with your Codat account team.
```
Comment (dcoplowe) [inline docs/integrations/accounting/xero/xero-app-partner-program.md:38]:
```suggestion
Xero may have a fee structure for your use case. For guidance on Xero fees, please speak to your account manager.
```

**Review by dcoplowe — COMMENTED** [review]
Minor changes

**Pull request merged**